src/components/styled.ts

```diff
@@ -52,5 +52,10 @@ export const iconCss = css`
   }
 `;
 
+export const greyButton = {
```

Comment:
what is `greyButton` in the terms of styles?

this file holds  general app-wide styles, not the component-specific